CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-08
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+
+* Features
+  * 
+
+* Preview Features
+  * 
+
+* Behavioral Changes
+  * With the [release](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#10042023--) of Container Insights 3.1.14, default 1-year tokens will be set to 1-hour expiry and refreshed at 10 minutes.
+  * A warning has been added for clusters utilizing dual-stack networking and outbound type user-defined routing if the associated route table does not have a default IPv6 route in place.
```

Comment:
How/Where is that warning displayed? At cluster create? 

Do we have an entry in the docs for this, @chasewilson

Reply:
Have a [PR here](https://github.com/MicrosoftDocs/azure-docs-pr/pull/254841) to put a note about default route. The error is just to let customers know as it's not a requirement and could be an issue for outbound traffic. Don't prevent customers from creating cluster as there are cases where IPv6 traffic is internal only and IPv4 is fine for outbound.

Reply:
Added link to the doc for detail reference